src/main/java/the_warlord/cards/warlord/TripleSlash.java

```diff
@@ -0,0 +1,51 @@
+package the_warlord.cards.warlord;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import the_warlord.WarlordMod;
+import the_warlord.actions.TripleSlashAction;
+import the_warlord.cards.CustomWarlordModCard;
+import the_warlord.characters.Warlord;
+
+
+public class TripleSlash extends CustomWarlordModCard {
+    public static final String ID = WarlordMod.makeID(TripleSlash.class.getSimpleName());
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Warlord.Enums.WARLORD_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int DAMAGE = 4;
+    private static final int UPGRADE_PLUS_DAMAGE = 1;
+    private static final int PLAY_COUNT = 2;
+
+    public TripleSlash() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = DAMAGE;
+        magicNumber = baseMagicNumber = PLAY_COUNT;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        addToBot(new DamageAction(m, new DamageInfo(p, damage), AbstractGameAction.AttackEffect.SLASH_HORIZONTAL));
+        for(int i = 1; i <= magicNumber; ++i){
+            // probably not the most elegant way to do this
+            addToBot(new TripleSlashAction(upgraded, m)); }
+    }
```

Comment:
There's a much easier way to do this, using CardMetaUtils::PlayCardAdditionalTime. It would look like this in this card's  use method:
        if (!this.purgeOnUse) {
            for (int i = 0; i < magicNumber; ++i) {
                CardMetaUtils.playCardAdditionalTime(this, m);
            }
        }

So no need for the custom action! The !purgeOnUse makes sure the temporary copies don't cause additional copies to be played.